resources/benchmark/tpcds/tables/call_center.csv.json

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
What are the jsons used for?

Reply:
Ah, sorry. I just re-read the PR description.

We will need the tpcds tools package anyway, correct? Should we include it as a submodule now and call dbgen from cpp (similar to what happens with the JOB)?

Reply:
done